src/runtime/node/inspector/index.ts

```diff
@@ -0,0 +1,36 @@
+// https://nodejs.org/api/inspector.html
+import noop from "../../mock/noop";
+import mock from "../../mock/proxy";
+import type inspector from "node:inspector";
+import type Console from "node:console";
+
+export const close: typeof inspector.close = noop;
+
+export const console: Console = mock.__createMock__("inspector.console");
+
+export const open: typeof inspector.open = () => ({
+  __unenv__: true,
+  [Symbol.dispose]() {
+    return Promise.resolve();
+  },
+});
```

Comment:
@jculvey are you happy with this simplification?

Reply:
Yup!